### DISCUSSION TITLE
Successfully deployed app won't display?

### DISCUSSION THREAD

**Discussion by john-mathis** [vercel/community#2024] [Help] [answered]
### Summary

My create-react-app is working just fine on localhost and It is successfully deploying on vercel. However, when I click the successful build it is showing the 404 error page. Code not found. 

### Additional information

Since it is successfully building and not throwing any errors, I'm not too sure what is causing the issue! 
I've tried changing the framework preset to create-react-app, adding adding a vercel.json file, and much more, but I still can't figure it out! 

Here is my package.json file
![image](https://user-images.githubusercontent.com/77464652/230799317-5f6a5bfa-4ca9-482e-9d27-c48255ca31be.png)

Here is my vercel.json file
![image](https://user-images.githubusercontent.com/77464652/230799279-bc357c66-17d5-4d53-ab08-06bf81bfcd69.png)

Here is my file tree
![image](https://user-images.githubusercontent.com/77464652/230799293-89a9af12-b756-4621-b255-fa1bd18ce630.png)

**Comment by amyegan** [comment, marked as answer]
It looks like you have a [monorepo](https://vercel.com/docs/concepts/monorepos) structure with the frontend code nested in a `client` directory. In that case, you may need to adjust the Root Directory in your project settings if you have not already done so. 

You should also check your Build & Development Settings. For most create-react-app sites, the "Create React App" preset is the appropriate option. But you can override the preset with custom commands if needed.

**Reply by amyegan** [reply]
Depending on your code, the vercel.json `rewrites` config may not be needed and could cause problems. If the first two suggestions don't solve it for you, you should look at our [Create React App template repo](https://github.com/vercel/vercel/tree/main/examples/create-react-app) for a simplified example

**Reply by john-mathis** [reply]
Thank you Amy!
